server/api/label.go

```diff
@@ -79,7 +79,7 @@ func (h *labelsHandler) GetStores(w http.ResponseWriter, r *http.Request) {
 
 	stores = filter.filter(stores)
 	for _, s := range stores {
-		store, err := cluster.TryGetStore(s.GetId())
+		store, err := cluster.GetStore(s.GetId())
```

Comment:
Personally, I prefer to return only singular `*StoreInfo` and change all `if err != nil` to `if store == nil`. Because it can only return 1 kind of error: the store does not exist.

Reply:
I can live with the current version too :smile_cat: